docs/Setup.md

```diff
@@ -157,7 +157,7 @@ The structure for both these properties are the same and are defined as below.
   "input": {
     "rootProperty": "rootProperty", // map incoming root property to an outgoing root property
     "rootObject.object1.object2.property": "rootObject.property", // map incoming nested property to an outgoing nested property
-    "array[]": "array[]", // map incoming root array to an outgoing array. Note: not specifying an index for the array will push in the new value, instead of overriding it at the specified index
+    "array": "array", // map incoming root array to an outgoing array.
```

Comment:
Is the brackets not needed for arrays? or only when referring to values inside of the array?

Reply:
the Bracket is not needed. With this ```"array[]": "array1[]"``` an input {array: [ 1, 2 ]} will map to array1: [[ 1, 2]]